src/models/streaming/SessionConfig.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Configuration for streaming job
+ */
+export class SessionConfig {
+    metadata?: string;
+
+    /**
+     * @param metadata (Optional) metadata to be associated with the streaming job
```

Comment:
Can the customer skip the constructor and just use an inline object?

Example:

`start({ metadata: "my metadata" })`?

Reply:
Yes they should be able to